docs/index-en.html

```diff
@@ -0,0 +1,4245 @@
+      <div id="toc"></div>
+      <!--INTRODUCTION SECTION-->
+      <div id="introduction">
+        <h2 id="intro" class="list">Introduction <span class="backlink"> back
+            to <a href="#toc">ToC</a></span></h2>
+        <span class="markdown">The OntoUML Vocabulary is a Web Ontology Language
+          (OWL) implementation of the <a href="http://w3id.org/ontouml/metamodel">OntoUML
+            Metamodel</a>. This vocabulary supports the serialization, exchange,
+          and publishing of OntoUML models as linked data.</span>
+        <div id="namespacedeclarations">
+          <h3 id="ns" class="list">Namespace declarations</h3>
+          <div id="ns" align="left">
+            <table>
+              <caption> <a href="#ns"> Table 1</a>: Namespaces used in the
+                document </caption>
+              <tbody>
+                <tr>
+                  <td><b>ontouml</b></td>
+                  <td><strong>&lt;https://w3id.org/ontouml#&gt;</strong></td>
+                </tr>
+                <tr>
+                  <td><b>w3id</b></td>
+                  <td>&lt;https://w3id.org&gt;</td>
+                </tr>
+                <tr>
+                  <td><b>dct</b></td>
+                  <td>&lt;http://purl.org/dc/terms&gt;</td>
+                </tr>
+                <tr>
+                  <td><b>owl</b></td>
+                  <td>&lt;http://www.w3.org/2002/07/owl&gt;</td>
+                </tr>
+                <tr>
+                  <td><b>rdf</b></td>
+                  <td>&lt;http://www.w3.org/1999/02/22-rdf-syntax-ns&gt;</td>
+                </tr>
+                <tr>
+                  <td><b>xml</b></td>
+                  <td>&lt;http://www.w3.org/XML/1998/namespace&gt;</td>
+                </tr>
+                <tr>
+                  <td><b>xsd</b></td>
+                  <td>&lt;http://www.w3.org/2001/XMLSchema&gt;</td>
+                </tr>
+                <tr>
+                  <td><b>rdfs</b></td>
+                  <td>&lt;http://www.w3.org/2000/01/rdf-schema&gt;</td>
+                </tr>
+                <tr>
+                  <td><b>dcat</b></td>
+                  <td>&lt;http://www.w3.org/ns/dcat&gt;</td>
+                </tr>
+                <tr>
+                  <td><b>vann</b></td>
+                  <td>&lt;http://purl.org/vocab/vann&gt;</td>
+                </tr>
+                <tr>
+                  <td><b>foaf</b></td>
+                  <td>&lt;http://xmlns.com/foaf/0.1&gt;</td>
+                </tr>
+                <tr>
+                  <td><b>sdo</b></td>
+                  <td>&lt;https://schema.org&gt;</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+      </div>
+      <!--OVERVIEW SECTION-->
+      <div id="overview">
+        <h2 id="overv" class="list">OntoUML Metamodel Vocabulary: Overview <span
+
+            class="backlink"> back to <a href="#toc">ToC</a></span></h2>
```

Comment:
Could we remove this links? They make the document cluttered.

Reply:
IMHO, they are important for the document navigation, especially considering the large amount of entities' declarations that we have. I vote to leave it as it is. It is working properly now that I added a table of contents.

If @tgoprince and @claudenirmf decide to remove, I won't oppose.

Reply:
I understand the function, but there are too many of them!

Reply:
Ok. Removed.